orca_detector/Dockerfile.dev

```diff
@@ -25,17 +25,42 @@ RUN pip install --upgrade setuptools
 RUN pip3 install jupyter -U && \
     pip3 install jupyterlab
 
+# Clone the repo
+RUN git clone https://github.com/paloukari/OrcaDetector
```

Comment:
Mike, I read through your comments. 

I'm using this container as my standalone development environment, the reason being, I need to cancel and re-provision a new IBM server for every coding session, my AWS credit is not approved yet. Because of that, It's nice to have all the setup and dependencies in the dockerfile.

If you don't want to change the way you work so far (I wouldn't if I were you), let's find a way that works for both of us. I think if you go ahead and create the container once and map the internal folders to the host, same as the previous mapping, you should still have the same experience as before. 
Lmk what you think.

Reply:
I thought about it again. I forgot that I was doing a clone on the host too. I'll change this back to volume mapping.

Reply:
I created a setup bash script that automates the setup process and reverted back to volumes mapping